From 43ae1b6340c8e6ff7d218f0fb89132e89b2d7b63 Mon Sep 17 00:00:00 2001
From: Matt Madison <matt@madison.systems>
Date: Wed, 18 Nov 2020 05:02:56 -0800
Subject: [PATCH 04/17] Restore version number to L4T builds

The stock makefiles omit the version number for L4T builds,
since they default to using a git hash that isn't available
in the L4T source distribution.

Since we already patch the makefiles to construct a version
number (from the L4T version), we can drop the conditional
that omits it, so it's easier to tell what version of cboot
is being used.

Signed-off-by: Matt Madison <matt@madison.systems>
---
 bootloader/partner/t18x/cboot/make/build.mk | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/bootloader/partner/t18x/cboot/make/build.mk b/bootloader/partner/t18x/cboot/make/build.mk
index 3053c7f..cd14f27 100644
--- a/bootloader/partner/t18x/cboot/make/build.mk
+++ b/bootloader/partner/t18x/cboot/make/build.mk
@@ -4,11 +4,7 @@ $(OUTBIN): $(OUTELF)
 	@echo generating image: $@
 	$(NOECHO)$(SIZE) $<
 	$(NOECHO)$(OBJCOPY) -O binary $< $@
-ifeq ($(NV_BUILD_SYSTEM_TYPE),l4t)
-	$(NOECHO)$(APPEND_VERSION) $(PROJECT) $(VERSION_MAX_LEN) $@
-else
 	$(NOECHO)$(APPEND_VERSION) $(BUILD_VERSION) $(PROJECT) $(VERSION_MAX_LEN) $@
-endif
 
 $(OUTELF).hex: $(OUTELF)
 	@echo generating hex file: $@
-- 
2.34.1

